### DISCUSSION TITLE
OpenDDSharp prequisites for Windows 10

### DISCUSSION THREAD

**Discussion by THCUser** [jmmorato/openddsharp#94] [General]
Hello,

I am trying to install offline prerequsites on operating system Windows 10 in order to communicate with an external client using openDDS sharp.
Do you know which prerequistes are mandatory ? and where I can find them ?
Thanks.

**Comment by jmmorato** [comment]
For runtime execution, you need to install the VC resdistributable: https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170

For communication, remember to open the windows firewall ports used for your domain.

**Comment by THCUser** [comment]
Thanks for your answer.
DO you know which ports are used in firewall ?
Because by default, we already open all ports in firewall, but it is not a secured way.

**Reply by jmmorato** [reply]
That depends of the domain you are using, the number of participants and the RTPS transport configuration.

RTI has a good article to explain that: https://community.rti.com/content/forum-topic/statically-configure-firewall-let-omg-dds-traffic-through
